data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -1430,17 +1432,55 @@
             y=13-19
         [/filter]
 
+#ifdef EASY
+        [scroll_to]
+            x,y=10,14
+        [/scroll_to]
+#else
         [terrain]
             terrain=Dd
             x=10,10,11,11,11,12,12
             y=14,15,15,16,17,15,16
         [/terrain]
-
+#endif
         {CHECK_EXPLORER}
+#ifdef EASY
+        [message]
+            speaker=$explorer.id
+            message= _ "Phew, thank goodness that this oasis is here! After hearing about the bandits in this area, I was beginning to worry that they had created a mirage to trick thirsty travelers such as ourselves!"
+        [/message]
+        [message]
+            speaker=Nym
+            message= _ "I know these bandits are evil, but do you really think they’d stoop so low as to get their victims’ hopes up before crushing them?"
+            image=portraits/nym_moody.png
+        [/message]
+        [message]
+            speaker=Zhul
+            message= _ "It has been known to happen before."
+        [/message]
+        [message]
+            speaker=Kaleh
+            message= _ "Truly, there are some depraved and cruel individuals in this world."
+        [/message]
+#else
         [message]
             speaker=$explorer.id
             message= _ "Dang. I was sure I saw an oasis here. Must have been a mirage. I’ve been out in the sand for too long."
         [/message]
+        [message]
+            speaker=Kaleh
+            message= _ "Perhaps it was the doing of those bandits..."
```

Comment:
I think it'd be best to leave out these explanations for the oasis. A big part of UtBS's story (and especially this scenario) is the natural hazards of a hard world. Replacing a natural mirage with a bandit-made trap detracts from that.

If the oasis vanishes, $explorer.id can say it must've been a mirage. If the oasis is there, $explorer.id can celebrate its presence.

Reply:
I added that part at the request of @ePaul's review comment above...

Reply:
I'm not seeing ePaul's comment, but in my opinion the event would be improved by removing all dialogue except _"Dang. I was sure I saw an oasis here. Must have been a mirage. I’ve been out in the sand for too long."_

Reply:
The [original discussion](https://github.com/wesnoth/wesnoth/pull/8421#discussion_r1588520489).

Before my comment, there was some dialog in the EASY mode for the purpose of lampshading "in the higher difficulty modes, the oasis might be a mirage". I said there should rather be something when there is actually a mirage, but accepted the argument that there should still be something when the oasis is real.

I agree with your argument that the whole "bandits might create mirages to lead travelers into traps" idea might be a bit strange, so maybe skipping this and just saying "I'm happy this is not a mirage" (indicating there actually might be (natural) mirages here) sounds better.

Reply:
Got it, thanks. I'm fine having easy-difficulty dialogue to lampshade the higher-difficulty mirages, like what you've suggested.

Reply:
ok, does [19e7a88](https://github.com/wesnoth/wesnoth/pull/8421/commits/19e7a88db350e45eb338cb178829ff5e3b782ea3) make it better?